Report.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PriceCalculator
+{
+    internal static  class Report
```

Comment:
Why use the ProductTax Class? Just use the taxService inside the report class and handle the logic being done in ProductTax inside Report instead. I dont see the need for ProductTax class.